docs/_source/guides/llms/practical_guides/collect_responses.md

```diff
@@ -81,12 +81,15 @@ If you feel that the disagreement between annotators is too high, especially for
 
 ## Solve disagreements
 
-In this section, we explore some techniques you can use to solve disagreements in the responses. These are not the only possible techniques and you should choose them carefully according to the needs of your project and annotation team.
+In this section, we explore some techniques you can use to solve disagreements in the responses. These are not the only possible techniques and you should choose them carefully according to the needs of your project and annotation team. Even though there are many ways in which you can unify responses, we offer support for some of them out-of-the box. There are two main ways to do this.
+
+```{include} /_common/tabs/unfication_strategies.md
+```
 
 Once you have unified your responses, you will have a dataset that"s ready for [fine-tuning](fine_tune.ipynb). Remember to save your unified dataset following one of the methods explained in [Export a Feedback dataset](export_dataset.ipynb).
 
+### Unifying RatingQuestions
```

Comment:
Maybe we could name this sub-section `### Unifying` (h3) and add `#### RatingQuestion` and the rest of the questions (h4), and just define the options as bullet points

Reply:
I agree